configuration/concepts/diags.csv

```diff
@@ -284,15 +293,15 @@ dbe756ab-0885-4d8f-a677-3587bb4a27fb,,ICD 10 - WHO:K12.1,"Oral ulcer, other form

```

Comment:
Here you add the UUID (I am guessing that the UUID of this concept in prod right?), but in fact do not modify the FSN of the `Offline Diagnosis` concept.
I don't think that adding the UUID here is needed. The initializer will be able to identify this concept, by FSN, and then modify its content just fine.